test/data/legacy_scatter.gslib

```diff
@@ -0,0 +1,9 @@
+Legacy GSLIB file
```

Comment:
Rename this file to `legacy_pset.gslib` and adjust the header like you did with the grid case:

```
# Legacy GSLIB point set file
```